tests/test_gam.py

```diff
@@ -22,6 +26,32 @@ def test_read_df_or_list():
     att_arr = np.asarray(df.values.tolist())
     feat_labels_arr = np.asarray(df.columns.tolist())
 
+    client = Client(memory_limit="auto")
+    ddf = dd.read_csv("tests/test_attributes.csv")
+    dask_att_arr = da.from_array(att_list)
+    dask_feat_labels_arr = da.from_array(feat_labels_list)
+
+    # Testing dask DataFrame
+    dask_df = gam.GAM(attributions=ddf)
```

Comment:
I would consider changing the name from `dask_df` to `g_ddf` otherwise this is confusing. Initially it made me think you are working with a dask dataframe and not the GAM object.